include/zephyr/logging/log_msg.h

```diff
@@ -226,20 +226,20 @@ enum z_log_msg_mode {
 #define Z_LOG_MSG_STACK_CREATE(_cstr_cnt, _domain_id, _source, _level, _data, _dlen, ...) \
 do { \
 	int _plen; \
-	uint32_t flags = Z_LOG_MSG_CBPRINTF_FLAGS(_cstr_cnt) | \
-			 CBPRINTF_PACKAGE_ADD_RW_STR_POS; \
```

Comment:
Side note: Originally I tried using "_flags" instead of "_options". However, this led to build warnings stating that "_flags" was initialised but not used. That warning is thought to come from a separate instance of "_flags" elsewhere in the chain of macros that comprise LOG_DBG().